src/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+}
+
+.card__image-block {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+  display: block;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__value {
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--black-color);
+}
+
+.card__description-title {
+  font-weight: 500;
+  font-size: var(--font-size-medium);
+  line-height: 18px;
+  margin: 40px 0 0;
+}
+
+.card__description-code {
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--light-gray);
+  margin-top: 4px;
+  margin-bottom: 16px;
+}
+
+.card__price-name {
+  font-size: var(--font-size-medium);
+  font-weight: 400;
+  color: var(--light-gray);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+}
+
+.card__price-value {
+  font-weight: 700;
+  font-size: 16px;
+  color: var(--black-color);
+}
+
+.card__button {
+  width: 100%;
+  background: var(--light-blue);
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  border: 1px solid var(--light-blue);
+  transition: 0.3s;
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated